win64svc/src/questdb.c

```diff
@@ -83,7 +83,6 @@ void buildJavaArgs(CONFIG *config) {
     LPCSTR javaOpts = "-XX:+UnlockExperimentalVMOptions"
                       " -XX:+AlwaysPreTouch"
                       " -XX:+UseParallelOldGC"
```

Comment:
This flag does not work on JDK 16:

![image](https://user-images.githubusercontent.com/7276403/111170965-6329ac80-859c-11eb-8960-1820c34555e1.png)